widget/embedded/src/Widget.tsx

```diff
@@ -1,4 +1,6 @@
 import type { WidgetConfig } from './types';
```

Comment:
This file is exporting 3 components, We can make it better.
We have a folder called `containers`, it can help us here.

Let's remove this file and make a separate component for each of these.
My suggestion:
`Main` to `/containers/App`
`Widget` -> `containers/Widget`
`WidgetState` -> `/containers/WidgetProvider`

Reply:
Note: I renamed `WidgetState` to `WidgetProvider` because it's more relavant to what we are actually doing here (providing a context).

Reply:
fixed.